packages/falso/src/lib/text.ts

```diff
@@ -1,105 +1,15 @@
ibus bibendum. Aliquam vel commodo velit, non vulputate erat. Nullam non aliquet nunc, eu aliquet dolor. Sed turpis dui, elementum a volutpat ut, varius a velit. Suspendisse et pulvinar enim. Integer est lectus, laoreet sed blandit sit amet, elementum ut ipsum. Aliquam varius, orci id fermentum maximus, erat nisi sagittis diam, id vestibulum tellus erat ultrices sapien. Donec et lorem magna. In euismod tellus at felis gravida commodo. Sed aliquet tristique aliquet. Cras mattis sed quam in rutrum. Donec at rutrum dolor, at luctus diam. Nulla dui purus, egestas nec urna at, sollicitudin maximus mi. Sed mi metus, ullamcorper ac odio eu, vulputate bibendum lorem. Aliquam at sapien sit amet massa dignissim aliquam eu sit amet urna. Maecenas nec elementum turpis, sed hendrerit turpis. Curabitur hendrerit lectus erat. Aliquam erat volutpat. In ultrices, nulla nec blandit dignissim, purus est pharetra mauris, non vestibulum orci lectus in lorem. Proin pellentesque magna at mi lacinia dictum. Suspendisse suscipit velit in felis interdum porttitor. Suspendisse lectus lorem, congue et sodales quis, porttitor venenatis nisl. Fusce ut cursus nulla. Morbi dignissim posuere tellus ac ultricies. Nunc in tincidunt enim. Duis iaculis porta pellentesque. Vivamus et faucibus turpis. Vivamus vestibulum fringilla velit. Nulla molestie, erat quis accumsan fermentum, dolor odio porta velit, nec interdum risus tellus quis mi. Morbi malesuada volutpat tellus, eu hendrerit odio convallis vitae. Quisque eleifend dolor quis ultrices euismod. Curabitur vulputate nec eros sed scelerisque. Praesent feugiat risus id libero cursus, ut ornare orci scelerisque. Ut cursus mattis orci, sed cursus diam sollicitudin vitae. Pellentesque pharetra fringilla orci, sed tristique tortor pretium in. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Pellentesque varius velit nec dolor dignissim finibus. Pellentesque et tellus vitae orci dapibus pellentesque. Nullam in sem ipsum. Etiam rhoncus erat tempus, semper tortor eu, cursus erat. Nulla at justo ut ante dignissim dignissim vel rutrum urna. Integer nulla massa, maximus nec pharetra vel, ultricies quis felis. Sed porttitor, augue eu molestie vestibulum, ipsum eros venenatis erat, vel luctus tortor mauris a mauris. Nullam non neque in metus euismod pulvinar. Pellentesque ac dolor bibendum, rutrum diam et, accumsan metus. Sed nisl ligula, tempus tempor nulla ac, fermentum tincidunt lorem. Integer lobortis.Quisque at nulla lorem. Maecenas ac feugiat metus, sed tincidunt tellus. Aenean quis elementum felis. Sed in venenatis massa. Vestibulum augue ipsum, rutrum sed massa non, convallis varius odio. Etiam iaculis tellus vel dolor molestie, vitae molestie ex ornare. Nulla suscipit, turpis sit amet bibendum eleifend, urna erat luctus mauris, sodales lacinia massa quam ac nunc. Nulla posuere gravida eros in rhoncus. Nunc porttitor ipsum venenatis varius mattis. In nec facil rosis metunc scelerisque magna necus.'
```

Comment:
I think it'll be better to use the `word` function here instead of this boilerplate text.

Reply:
@NetanelBasal that's fair. I'm not too happy with the static text either, but I'm not sure about the word function.
How are you imagining it? Do we create a loop adding words and spaces until the char length is met? Im not sure if that might be a little inefficient. 
It would also be more realistic if we could sprinkle punctuation throughout.

Reply:
I don't think there will be a performance issue. Take the time to think about a nice solution, no rush.

Reply:
@NetanelBasal I've pushed an update with an idea I had while eating my breakfast. I've added an array of sentence structures to build the text from. 

What are your thoughts?